### PR TITLE
Updated calibration UI visuals and added UserActivity events.

### PR DESCRIPTION
**Pull request by druiz17**
updated calibration UI visuals and added UserActivity events.

### CODE REVIEW

**Comment by druiz17** [comment]
Test Plan
- there should be no functional changes.
- make some UI changes with calibration
- if you check http://metabase.highfidelity.com you can check for user events like
mocap_ui_open_dialog
mocap_ui_close_dialog
mocap_ui_failed
mocap_ui_success
mocap_button_fail
mocap_button_success
mocap_button_uncalibrate

**Comment by anshuman64** [comment]
Alan and I will test this

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1187/HighFidelity-Beta-PR10849-f077e40dd3f7ec73973bf9b1134eb3857016ad3a.exe

**Comment by druiz17** [comment]
test this please

File: plugins/openvr/src/ViveControllerManager.cpp [hunk 1]
```diff
@@ -374,27 +376,21 @@ QJsonObject ViveControllerManager::InputDevice::configurationSettings() {
...
-    } else if (!_calibrated && success) {
-        status["calibrated"] = _calibrated;
-        status["success"] = success;
-        status["configuration"] = configToString(_preferedConfig);
-        status["puckCount"] = (int)_validTrackedObjects.size();
-    }
+    status["calibrated"] = _calibrated;
+    status["configuration"] = configToString(_preferedConfig);
+    status["head_puck"] = (_headConfig == HeadConfig::Puck) ? true : false;
+    status["hand_pucks"] = (_handConfig == HandConfig::Pucks) ? true : false;
```
Comment (sethalves) [inline plugins/openvr/src/ViveControllerManager.cpp]: these two conditional-expression operators are redundant.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1205/HighFidelity-Beta-PR10849-f077e40dd3f7ec73973bf9b1134eb3857016ad3a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1205/HighFidelity-Beta-PR10849-f077e40dd3f7ec73973bf9b1134eb3857016ad3a.dmg

**Comment by anshuman64** [comment]
Looks like some data is coming in, waiting for data points from my test today to come in to confirm.

Also, one UI change that I forgot Alan mentioned: some text on the pages for plugins that have no configurations to change. We can ask Alan what the text exactly should be.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1237/HighFidelity-Beta-PR10849-a749fa19adb3deced0945199505382bfc5f5a67e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1237/HighFidelity-Beta-PR10849-a749fa19adb3deced0945199505382bfc5f5a67e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1282/HighFidelity-Beta-PR10849-5fbcd494be53abca97c18cfad02ceae3c3469c13.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1282/HighFidelity-Beta-PR10849-5fbcd494be53abca97c18cfad02ceae3c3469c13.exe

**Pull request merged**